pennylane/_device.py

```diff
@@ -422,8 +431,8 @@ def var(self, observable, wires, par):
         """
         raise NotImplementedError("Returning variances from QNodes not currently supported by {}".format(self.short_name))
 
-    def sample(self, observable, wires, par, n=None):
-        """Return a sample of an observable.
+    def sample(self, observable, wires, par):
+        """Return a sample of an observable with a size of ``device.shots``.
```

Comment:
```suggestion
        """Return a sample of an observable.
        
         The number of samples is determined by the value of ``device.shots``,
         and can be modified via...
```

Reply:
Note: don't commit the above until the `...` is expanded!

Reply:
``device.shots`` doesn't actually point anywhere. Should it be ``self.shots`` or ``Device.shots``?